components/auth/authRoutes.js

```diff
@@ -25,24 +27,37 @@ router.get(
   passport.authenticate('google', { failureRedirect: '/' }),
   controllers.googleHandler
 )
-/*  ================== MEETUPS ================== */
 
+/*  ================== MEETUPS ================== */
 router.get('/meetup', passport.authenticate('meetup'))
 
 router.get(
   '/meetup/cb',
   passport.authenticate('meetup'),
   controllers.meetupHandler
 )
-/*  ================== GOODREADS ================== */
 
+/*  ================== GOODREADS ================== */
 router.get('/goodreads', passport.authenticate('goodreads'))
 
+// allow developer to generate a token 4 REST client usage
+if (process.env.NODE_ENV === 'development') {
+  router.get('/token', async (req, res) => {
+    /*  ================== Make sure to run the seeds! ================== */
+    const user = await db('users')
+      .where({ id: 1 })
+      .first()
```

Comment:
```suggestion
    const user = await db('users')
      .where({ id: process.env.DEV_USER_ID })
      .first()
```

@luiscmartinez 
Don't we want this to be dynamic? So we can test via postman using our own dev user.  `user_id = 1` will be a fake user.